### PR TITLE
update-window.sh should handle both etcd and etcd2 installation.

### PR DESCRIPTION
**Pull request by xueshanf**
Right now the update-window.sh won't work properly on etcd2 installation - it would just randomly reboot etcd2 members without reboot locking.

### CODE REVIEW

File: os/update-strategies.md [hunk 1]
```diff
@@ -148,24 +148,40 @@ This script should be installed in a location to match the script path used in t
 
 ```yaml
```
Comment (joshix) [inline os/update-strategies.md]:
```sh

**Comment by joshix** [comment]
Note: @kayrus is working on a different approach to this concept.

**Comment by crawford** [comment]
Locksmith 0.3.4 has support for reboot windows. Once this hits Stable (should be next week), we will no longer need this workaround.

**Comment by kayrus** [comment]
@joshix not yet.

**Comment by joshix** [comment]
> Locksmith 0.3.4 has support for reboot windows. Once this hits Stable (should be next week), we will no longer need this workaround.

Has this become true?

**Comment by crawford** [comment]
> Has this become true?

Yes.

**Comment by joshix** [comment]
Closing in favor of stock reboot window support in later versions of locksmith (0.3.4+)

**Pull request closed without merging**